### ISSUE TITLE
Secondary driver prevents sway from launching

### ISSUE THREAD

**Issue by vleni** [WillPower3309/swayfx#223]
### Please read the following before submitting:
- Proprietary graphics drivers, including nvidia, are not supported. Please use the open source equivalents, such as nouveau, if you would like to use Sway.
- Please do NOT submit issues for information from the github wiki. The github wiki is community maintained and therefore may contain outdated information, scripts that don't work or obsolete workarounds.
  If you fix a script or find outdated information, don't hesitate to adjust the wiki page.

### Please fill out the following:
- **Swayfx Version:**
  - `sway -v`
```
sway version 0.3.2
```

- **Debug Log:**
  - Run `sway -d 2> ~/sway.log` from a TTY and upload it to a pastebin, such as gist.github.com.
  - This will record information about sway's activity. Please try to keep the reproduction as brief as possible and exit sway.
  - Attach the **full** file, do not truncate it.
```
00:00:00.000 [INFO] [sway/main.c:338] Sway version 0.3.2
00:00:00.000 [INFO] [sway/main.c:339] wlroots version 0.16.2
00:00:00.004 [INFO] [sway/main.c:120] Linux Mellite 6.4.12-arch1-1 #1 SMP PREEMPT_DYNAMIC Thu, 24 Aug 2023 00:38:14 +0000 x86_64 GNU/Linux
00:00:00.004 [INFO] [sway/main.c:136] Contents of /etc/lsb-release:
00:00:00.004 [INFO] [sway/main.c:120] DISTRIB_ID="Arch"
00:00:00.004 [INFO] [sway/main.c:120] DISTRIB_RELEASE="rolling"
00:00:00.004 [INFO] [sway/main.c:120] DISTRIB_DESCRIPTION="Arch Linux"
00:00:00.004 [INFO] [sway/main.c:136] Contents of /etc/os-release:
00:00:00.004 [INFO] [sway/main.c:120] NAME="Arch Linux"
00:00:00.004 [INFO] [sway/main.c:120] PRETTY_NAME="Arch Linux"
00:00:00.004 [INFO] [sway/main.c:120] ID=arch
00:00:00.004 [INFO] [sway/main.c:120] BUILD_ID=rolling
00:00:00.004 [INFO] [sway/main.c:120] ANSI_COLOR="38;2;23;147;209"
00:00:00.004 [INFO] [sway/main.c:120] HOME_URL="https://archlinux.org/"
00:00:00.004 [INFO] [sway/main.c:120] DOCUMENTATION_URL="https://wiki.archlinux.org/"
00:00:00.004 [INFO] [sway/main.c:120] SUPPORT_URL="https://bbs.archlinux.org/"
00:00:00.004 [INFO] [sway/main.c:120] BUG_REPORT_URL="https://bugs.archlinux.org/"
00:00:00.004 [INFO] [sway/main.c:120] PRIVACY_POLICY_URL="https://terms.archlinux.org/docs/privacy-policy/"
00:00:00.004 [INFO] [sway/main.c:120] LOGO=archlinux-logo
00:00:00.004 [INFO] [sway/main.c:108] LD_LIBRARY_PATH=
00:00:00.004 [INFO] [sway/main.c:108] LD_PRELOAD=
00:00:00.004 [INFO] [sway/main.c:108] PATH=/home/aniva/.local/bin:/home/aniva/.elan/bin:/usr/local/sbin:/usr/local/bin:/usr/bin:/usr/bin/site_perl:/usr/bin/vendor_perl:/usr/bin/core_perl
00:00:00.004 [INFO] [sway/main.c:108] SWAYSOCK=/run/user/1000/sway-ipc.1000.2610.sock
00:00:00.005 [ERROR] [sway/main.c:65] Proprietary Nvidia drivers are NOT supported. Use Nouveau. To launch sway anyway, launch with --unsupported-gpu and DO NOT report issues.`
```

- **Configuration File:**
  - Please try to produce with the default configuration.
  - If you cannot reproduce with the default configuration, please try to find the minimal configuration to reproduce.
  - Upload the config to a pastebin such as gist.github.com.

The default configuration is sufficient

- **Description:**
  - The steps you took in plain English to reproduce the problem.

1. Install both nvidia and AMD graphics cards on a machine
2. Connect the monitor to the AMD graphics card
3. Install open source mesa driver for AMD but proprietary driver for nvidia
4. Launch sway, and it says
```
00:00:00.005 [ERROR] [sway/main.c:65] Proprietary Nvidia drivers are NOT supported. Use Nouveau. To launch sway anyway, launch with --unsupported-gpu and DO NOT report issues.
``` 
In principle it shouldn't matter because the nvidia cards are not used for display

**Comment by ErikReider** [comment]
This is an upstream choice/issue. Please open an issue in the upstream [sway](https://github.com/swaywm/sway) repo.

Also, test base sway-git and see if anything has changed :)
